avro/src/test/scala/magnolify/avro/test/AvroTypeSpec.scala

```diff
@@ -149,6 +134,22 @@ object AvroTypeSpec extends MagnolifySpec("AvroType") {
     expectException[IllegalArgumentException](AvroType[DoubleFieldDoc]).getMessage ==
       "requirement failed: More than one @doc annotation: magnolify.avro.test.DoubleFieldDoc#i"
   )
+
+  {
```

Comment:
Not pretty but passable without pulling in ScalaTest which I've been punting on forever :shrug:
Also more explicitly checking the field names.